### PR TITLE
add headers to Limerick file

### LINKED ISSUE #20: darrenob tracks progress on GitHub flow
- [x] - create a branch (git)
- [x] - add some *commits* (git)
- [x] - open a pull request (GitHub)
- [ ] - additional commits (git)
- [ ] - merge (git)

### PR DESCRIPTION
**Pull request by darrenob**
This PR adds a Limerick file to the class repo. This PR closes #20

### CODE REVIEW

**Comment by darrenob** [comment]
@vsabarly do you know of any good clubs in Barcelona?

File: darrenob-limerick.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Limerick, IE
+### Restaurants
+- this
+- that
+### Attractions for Tourists
+- castle
+- pubs
+### Attractions for Locals
```
Comment (vsabarly) [inline darrenob-limerick.md]: What about the clubs in Limerick :p

**Comment by vsabarly** [comment]
Great commit! Go to Apolo man.

File: darrenob-limerick.md (hunk 1 above)
Comment (darrenob) [inline darrenob-limerick.md]: @vsabarly I'll get back to you on that

**Comment by vsabarly** [comment]
👍  we're all waiting for you Darren

**Comment by neonblueflame** [comment]
:+1:

**Pull request merged**